packages/terra-base/src/Base.scss

```diff
@@ -4,16 +4,16 @@
 // More info: https://css-tricks.com/box-sizing/#article-header-id-6
```

Comment:
+1 I added this to our discussion on Friday. I don't want this to block your review.